component-maps.yml

```diff
@@ -369,6 +376,13 @@ docker_image:
     - mender-deviceconnect
     release_component: true
 
+  devicemonitor:
+    git:
+    - devicemonitor
+    docker_container:
+    - mender-devicemonitor
+    release_component: true
```

Comment:
Here and below should be false

Reply:
arrrgh! sorry! copy paste and in a hurry mistake.  shame on me.